skins/shared/styles/buttons/buttons-wikia.scss

```diff
@@ -9,10 +9,10 @@ a.wikia-button,
 .wikia-single-button a,
 .wikia-menu-button,
 input[type="submit"]:not(.wds-button),
```

Comment:
This one also needs to have :not(.wds-button) removed to be input[type="submit"] since wds-buttons have been fixed to not conflict with the 2010 era buttons and can also fix button styling on wikis with customized one such as http://el.ikariam.wikia.com which has input[type="submit"] customized.